worksheet_A/TerminalHacking.cpp

```diff
@@ -3,9 +3,32 @@
 
 #include "stdafx.h"
 #include "WordList.h"
+#include <string>
+#include <iostream>
+
+
+using namespace std;
 
 const int wordLength = 5;
 const int numberOfWords = 15;
+int PlayerLives = 4;
+int Likeness = 0;
+string Guess = "";
```

Comment:
Do these need to be global variables? might be better to declare them inside `main()`, close to the point of use.